docs/deployment/docker-setup.md

```diff
@@ -1,9 +1,13 @@
 # Docker
-## What is docker
-TODO
+## Docker in really short
+In really short, docker is like a virtual machine but without the kernel emulation. Every container share the host kernel. For rtc, we use linux containers that require a linux kernel.
```

Comment:
Linux (with capital L)

Reply:
I think better will be: 'In short, Docker is like a virtual machine but without kernel emulation' (as soon as I consider my English :)

Reply:
In next commit